src/main/java/dev/aura/bungeechat/listener/BungeeChatEventsListener.java

```diff
@@ -16,33 +17,31 @@
 import net.md_5.bungee.event.EventPriority;
 
 public class BungeeChatEventsListener implements Listener {
-  private static final List<UUID> joinedPlayers = new LinkedList<>();
-  private static final List<UUID> duplicatePlayers = new LinkedList<>();
-
-  @EventHandler(priority = EventPriority.HIGHEST)
-  public void onPlayerJoin(PostLoginEvent e) {
```

Comment:
Why did you remove the player join event?

Note im currently on mobile so only pointing out the glaringly obvious things. I might have missed a thing or two.

Reply:
`PostLoginEvent` was used only to check for duplicate connections. I've moved this check to `ServerSwitchEvent`. `BungeeChatJoinEvent` was and still is called from `ServerSwitchEvent`.